### PR TITLE
Use shared github action from fog

### PR DESCRIPTION
**Pull request by dosas**

### CODE REVIEW

**Comment by dosas** [comment]
@ShimShtein A review would be highly appreciated.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
```
Comment (ShimShtein) [inline .github/workflows/ci.yml:8]: Why do we want to enable the tests only on schedule? Is there any condition that prevents us from running tests on each PR and push to master?

Comment (dosas) [reply .github/workflows/ci.yml:8]: The tests are exectued already on push to master. I removed the branch from pull_request, so it should run for all PRs

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
@dosas I have merged the PR, but I see the tests are failing: https://github.com/fog/fog-ovirt/actions/runs/10486087079/job/29043563313#step:3:110
Can you please take a look? Now at least we will be  able to see the tests on the next PR :)

**Comment by dosas** [comment]
> @dosas I have merged the PR, but I see the tests are failing: https://github.com/fog/fog-ovirt/actions/runs/10486087079/job/29043563313#step:3:110 Can you please take a look? Now at least we will be able to see the tests on the next PR :)

Yes, I saw that too. Sadly this is due to missing libcurl on githubs ubuntu runners. I'am not sure if this can be easily fixed or if the template needs to be adapted.

**Comment by ShimShtein** [comment]
We can always run it in a container :shrug:

**Pull request merged**